lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm

```diff
@@ -153,8 +155,10 @@ sub create {
     my $self = $class->SUPER::create(@_);
     return unless ($self);
 
-    my $rv = eval {
+    $self->_lock_per_lane_alignment();
+    my @temp_allocations = ();
 
+    try {
         #TODO In a future version collect relevant alignments from other merged alignment results when available
```

Comment:
Ah! That makes sense. That would be really nice, especially with the way things are now with needing to recreate per-lane bams just to do a new merge. I don't know that we'll be able to get to this immediately but I'll certainly keep it in mind for future alignment work.